docs/en/sql-reference/statements/show.md

```diff
@@ -6,14 +6,16 @@ sidebar_label: SHOW
 
 # SHOW Statements
 
-## SHOW CREATE TABLE
+## SHOW CREATE TABLE or DATABASE
 
 ``` sql
-SHOW CREATE [TEMPORARY] [TABLE|DICTIONARY|VIEW] [db.]table|view [INTO OUTFILE filename] [FORMAT format]
+SHOW [CREATE] [TEMPORARY] [TABLE|DICTIONARY|VIEW|DATABASE] [db.]table|view [INTO OUTFILE filename] [FORMAT format]
 ```
 
 Returns a single `String`-type ‘statement’ column, which contains a single value – the `CREATE` query used for creating the specified object.
 
+`SHOW TABLE t`, `SHOW CREATE t` and `SHOW DATABASE db` have the same meaning as `SHOW CREATE TABLE|DATABASE t|db`, but `SHOW t` and `SHOW db` are not supported.
+
```

Comment:
About `SHOW CREATE` (see the discussion in 02710_show_table.sql):

It's okay with keeping the semantics as is. But I also think that the `SHOW CREATE <name>` syntax is weird because 1. its ambiguity 2. getting it right means it also needs to support RBAC types i.e. `SHOW CREATE user/role/row policy/quota/settings profile`, 3. the existing tests  hardly use it 4. MySQL also doesn't support it (try `drop table tab;create table tab(a int);show create tab;` at https://extendsclass.com/mysql-online.html). So, due to backward compatibility we can't get rid of `SHOW CREATE` but maybe we can "un-emphasize" it in the docs? For example, we could
- remove the `[` and `]` around `TABLE|DICTIONARY|VIEW|DATABASE` in l. 12
- say in l. 17 just "Keyword `CREATE` is optional and can be omitted."

(btw., l. 9 should be `### SHOW CREATE TABLE | DICTIONARY | VIEW | DATABASE`)

Reply:
OK, I agree with it.